src/main/java/nextstep/ladder/RandomLineStrategy.java

```diff
@@ -0,0 +1,21 @@
+package nextstep.ladder;
+
+import java.util.Random;
+
+public class RandomLineStrategy implements LineStrategy{
+    private Random random = new Random();
+
+    public RandomLineStrategy(){
+    }
+    public RandomLineStrategy (Random random){
+        this.random = random;
+    }
+
+    @Override
+    public boolean drawLine(boolean lastLine) {
+        boolean line = random.nextBoolean();
+        if(!lastLine && line)
+            return true;
+        return false;
```

Comment:
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)에 맞춰서 괄호를 추가해 주시면 좋을 것 같아요 
if condition도 조금 간단하게 표현할 수 있을 것 같아요
```suggestion
        if (!lastLine) {
            return random.nextBoolean();
        }
        return false;
```